AlignCommerce/code-sample.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+  let colorsUsed = {};
+  function getNewRandomColor() {
+    let newColor = '';
+    do {
+      let r = Math.floor(Math.random() * 255);
+      let g = Math.floor(Math.random() * 255);
+      let b = Math.floor(Math.random() * 255);
+      newColor = `${r},${g},${b}`;
+    } while (colorsUsed[newColor] == true);
+
+    return newColor;
+  }
+
+  let body = document.getElementsByTagName('body')[0];
+  function changeColor() {
+    let newColor = getNewRandomColor();
+    body.style.backgroundColor = `rgb(${newColor})`;
+    colorsUsed[newColor] = true;
+
+    if(Object.keys(colorsUsed).length < 10) {
```

Comment:
can we make this more flexible in case the product specs changes, e.g (10.72 seconds)